input/fsh/moped-hauptversicherter.fsh

```diff
@@ -0,0 +1,4 @@
+Profile: Hauptversicherter
+Parent: HL7ATCorePatient
```

Comment:
Hauptversicherter sollte auf den ersten Blick eine Person Ressource sein.

Reply:
da müssen wir noch mit der SVC sprechen, damit das dann auch in Coverage als solches abgebildet wird (subscriber)